docs/.vuepress/config.js

```diff
@@ -1,4 +1,6 @@
-module.exports = _ctx => ({
+module.exports = (_ctx) => ({
+  sourceDir: 'docs',
```

Comment:
Note that the source directory is now specified in the config as opposed to an inline argument.